doc/bluealsa-aplay.1.rst

```diff
@@ -173,6 +253,9 @@ Alternatively we can define a PCM with the required setting:
         }
     }
 
+EXAMPLES
+========
+
```

Comment:
> I would use --mixer-device=hw:USB to be consistent with the pcm naming. Also, it might be useful to preface this with a note that these names are just examples.

Yes, that will be better indeed. But (and here is the limit of my ALSA knowledge) how to list available mixer names? In `alsamixer` the case is simple, there is a combobox with available hardware mixers ....but `amixer` can not handle these names as a valid mixer device. Or simply mixer device is in fact a card name?

> A list of control names for a card called "USB" can be obtained by running `amixer -c USB scontrols`

OK, such info could be useful.

Reply:
> amixer can not handle these names as a valid mixer device

So, amixer (and alsamixer) can be given a mixer either by using the card name as in `-c NAME`, which is clearly only valid for sound cards, or by using the device name, as in `-D DEVICE_NAME`. A device name is a configuration entry such as `hw:CARD=USB`, `bluealsa:DEV=00:00:00:00:00:00`, etc, with the usual argument expansion. Clearly device name is more general, but is also *much* more difficult to explain to ALSA novices. There is no utility from the ALSA project that lists CTL devices on the command line. I have my own here: https://github.com/borine/alsa-utils-completions because I was unable to find any ready-made example.

Reply:
I've added a little bit rephrased your suggestion as a preface to command line examples. So, right now in the example we've got mentioned `aplay -l` which might be a good "tip" for a newbie to ALSA as well.

Reply:
That looks good to me. FWIW card names can also be obtained by
```
cat /proc/asound/cards
```
or
```
cat /sys/class/sound/card*/id
```
of which the second gives a simple list of names with no need to parse the output, but also no clue as to which name refers to which card .